apps/studio/prisma/scripts/moh-tosp/createCollectionFromLocal.ts

```diff
@@ -0,0 +1,155 @@
+import fs from "fs/promises"
+import path from "path"
+
+import { db, jsonb } from "~/server/modules/database"
+import { FileLogger } from "../FileLogger"
+
+// Update the logger path if required
+const logger = new FileLogger("./createCollectionFromLocal.log")
+
+export const createCollectionFromLocal = async (
+  contentDir: string,
+  siteId: number,
+  indexPageName: string, // should be placed outside the folder
```

Comment:
not sure if i understand why this should be placed outside the folder?

Reply:
This follows the same structure as what github based repos have due to filesystem based routing, the index page is outside the children folder

e.g. /hello-world.json -> index page
    /hello-world/ -> folder